Marlin/src/gcode/bedlevel/mbl/G29.cpp

```diff
@@ -85,7 +85,7 @@ void GcodeSuite::G29() {
       mbl.reset();
       mbl_probe_index = 0;
       if (!ui.wait_for_bl_move) {
-        queue.inject_P(PSTR("G28\nG29 S2"));
+        process_subcommands_now_P(PSTR("G28\nG29 S2"));
```

Comment:
You would never use the `G29` from MBL in a G-code macro, so the `inject_P` is perfectly fine here.